fcrepo-indexing-solr/src/main/java/org/fcrepo/camel/indexing/solr/SolrRouter.java

```diff
@@ -94,7 +95,7 @@ public void configure() throws Exception {
             .filter(not(or(header(IDENTIFIER).startsWith(simple("{{audit.container}}/")),
                     header(IDENTIFIER).isEqualTo(simple("{{audit.container}}")))))
             .to("fcrepo:{{fcrepo.baseUrl}}?preferOmit=PreferContainment")
-            .setHeader(FCREPO_TRANSFORM).xpath(hasIndexingTransform, String.class, ns)
+            .setHeader(FCREPO_TRANSFORM).xpath(hasIndexingTransformation, String.class, ns)
```

Comment:
The problem was that the fixture used the same incorrect property, and there is already a unit test for this. Short of validating against the indexing ontology (which is where this is defined), I'm not sure that an additional integration test would do anything for us.